sympy/stats/crv_types.py

```diff
@@ -710,6 +711,7 @@ def _moment_generating_function(self, t):
     def _quantile(self, p):
         return self.x0 + self.gamma*tan(pi*(p - S.Half))
 
+
```

Comment:
Can we avoid these new line changes? As it seems a bit irrelevant for this PR. May be someone else can do it in a different PR specifically aiming for styling the code correctly.

Reply:
A leftover of adding methods and later removing them. I don't see any problem with this edit, standardizes to PEP-8 format and does not inhibit the readability of the changes of this PR. (People reading this PR can just skip through these lines).